docs/mkdocs.yml

```diff
@@ -8,14 +8,19 @@ edit_uri: https://github.com/hedgedoc/hedgedoc/edit/master/docs/content/
 nav:
   - Home: index.md
   - Installation:
-      - 'Manual Installation': setup/manual-setup.md
-      - 'Reverse Proxy': setup/reverse-proxy.md
-      - Docker: setup/docker.md
-      - Cloudron: setup/cloudron.md
-      - Heroku: setup/heroku.md
-      - LinuxServer: setup/docker-linuxserver.md
-      - Yunohost: setup/yunohost.md
+      - Official installation methods:
+        - Docker: setup/docker.md
+        - Heroku: setup/heroku.md
+        - 'Manual Installation': setup/manual-setup.md
```

Comment:
I'd suggest switching Heroku and Manual Installation here

Reply:
done